ooniprobe/Base.lproj/Localizable.strings

```diff
@@ -63,12 +95,14 @@
 "laws_text_6" = "To our knowledge, no ooniprobe user has ever faced consequences from the use of our platform. However, many countries have a lengthy history of targeting and abusing digital rights activists, which could leave ooniprobe users subject to severe civil, criminal, or extra-judicial penalties.";
 "i_understand_the_risks" = "I understand the risks.";
 
+/*
```

Comment:
👎

Reply:
As said, it's not a merge in master, every old string will be kept here until final

Reply:
I don't know if they will be used or not. As we don't even have definitive strings for now.
I want to touch them only at the end because there are many strings file to edit.
For now I'm just commenting every portion of strings to specify where they will be used, when the app is done all the unused strings will be removed.
If you don't like commented strings I can uncomment them.

Reply:
But these strings are duplicated above, so you should just delete them as they already have an alternative key for them.

Also I see you have `true` and `false` that is redefined as `_true` and `_false`, but it's actually the same thing.

Trust me, git will not loose your strings :)

Reply:
And what should I do with the nonEN strings?